### DISCUSSION TITLE
Add display filtering (exclusions) of audio devices shown in list

### DISCUSSION THREAD

**Discussion by gmckeown** [File-New-Project/EarTrumpet#678] [Ideas]
I have a lot of audio devices that I don't often use; e.g. the Steam streaming microphone and speakers. Even when I do use them, I don't necessarily need to be able to change the volume using EarTrumpet, such as if streaming a Steam game to a TV and can use the volume controls there.

I think it would be useful to have a display filter so that I could specify a few regexes to exclude certain devices from management by Eartrumpet, e.g.:

```
^.*Steam Streaming.*$
^.*NoMachine.*$
^DELL
U2414H
```
(I know some of those patterns could be simpler, but wanted to highlight the fact that they're regexes.)

A comment in issue #235 suggested disabling devices that aren't actively used. That's fine for devices that are *never* used, but for those that are occasionally needed, the process of enabling via Device Manager is a little bit of a pain. That said, I disabled a bunch of my devices:

![image](https://user-images.githubusercontent.com/22765233/51174847-b0bd0880-18b0-11e9-9c25-3251d2935819.png)

These devices are still appearing in EarTrumpet. Have exited and restarted the app, which hasn't made a difference:

![image](https://user-images.githubusercontent.com/22765233/51174332-7606a080-18af-11e9-88cd-07702674e780.png)

Haven't rebooted yet, but this is my work machine and I need to avoid killing some remote sessions at the moment. Of course, if it's necessary to reboot to refresh the device list, that's even *more* of a pain! ;)

**Comment by riverar** [comment]
Yup, our machines look like this too. We're working on it!

**Comment by vertigo220** [comment]
One of the issues I have as well. I was going to suggest even just allowing each audio device to be collapsed/expanded individually, instead of either showing only the current one or all of them. So when first opening it, show current, when expanding that, show all but show them in their last expanded/collapsed state.

**Comment by Nilzor** [comment]
Any update on this since "we're working on it" Jan 2019?

**Comment by rcmaehl** [comment]
Joining in. Any updates?
